torch/lib/c10d/reducer.cpp

```diff
@@ -988,6 +988,12 @@ void Reducer::prepare_for_forward() {
   std::lock_guard<std::mutex> lock(mutex_);
   num_iterations_++;
   forward_start_time_ = current_time_in_nanos();
+  // Log runtime (e.g. avg performance) stats at randomly selected iterations.
+  if (num_iterations_ == 10
```

Comment:
Can we insert these iterations into a constant set so that they can be easily modified and extended as needed?

Reply:
sounds good